gradio/inputs.py

```diff
@@ -63,6 +63,11 @@ def get_shortcut_implementations(cls):
         """
         return {}
 
+    def rebuild_flagged(self, dir, msg):
```

Comment:
We probably want to use this method for more than just flagging. Any time we have an image / sample in the front end, and we want to convert it back to a Python object, we should use this method. So I actually think a better name for this might just be `rebuild` or `pythonify` or `deprocess`

Reply:
changed name to `rebuild`